dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops

```diff
@@ -237,3 +237,7 @@ info.directory.app.useraction=No action is required.
 info.unexpanded.app=CWWKZ0136I: The {0} application is using the archive file at the {1} location.
 info.unexpanded.app.explanation=The location for the application points to a binary archive file. The archive file will not be expanded on disk because automatic expansion of applications was not configured.
 info.unexpanded.app.useraction=No action is required.
+
+warning.could.not.expand.app.loc=CWWKZ0137W: Could not expand the application {0} at the location specified in expandLocation {1}.
+warning.could.not.expand.app.loc.explanation=The application could not be expanded to the location specified in expandLocation. The default location ${server.config.dir}/apps/expanded/ was used for the application. 
+warning.could.not.expand.app.loc.useraction=Correct the value specified in expandLocation and either restart the server or update the application.
```

Comment:
suggested edit:

Correct the value that is specified by the expandLocation property and either restart the server or update the application.

Reply:
updated

Reply:
I see the fixes in the first two but not in `useraction`

Reply:
Sorry about that..  Should be there now.